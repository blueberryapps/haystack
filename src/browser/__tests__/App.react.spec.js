import App from '../App.react';
import { shallow } from 'enzyme';
import React from 'react';

test('shallowly renders App', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
