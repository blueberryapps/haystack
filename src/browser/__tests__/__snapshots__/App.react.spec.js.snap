exports[`test shallowly renders App 1`] = `
<div>
  <HelmetWrapper
    title="Haystack" />
  <button
    onClick={[Function]}>
    Haystack
    <img
      alt="logo"
      src="test-file-stub" />
  </button>
</div>
`;
